openstack/nova/templates/hypervisors/hypervisors-vmware-secret-vct.yaml

```
@@ -0,0 +1,35 @@
+{{- if or (.Capabilities.APIVersions.Has "vcenter-operator.stable.sap.cc/v1") (.Values.isImageTransportTemplating | default false) }}
+apiVersion: vcenter-operator.stable.sap.cc/v1
+kind: VCenterTemplate
+metadata:
+  name: 'vcenter-cluster-nova-compute-secret'
+options:
+  scope: 'cluster'
+  jinja2_options:
+    variable_start_string: '{='
+    variable_end_string: '=}'
+template: |
+  apiVersion: v1
+  kind: Secret
+  metadata:
+    name: nova-compute-vmware-{= name =}
+    labels:
+      system: openstack
+      type: configuration
+      component: nova
+      vcenter: {= host =}
+      datacenter: {= availability_zone =}
+      vccluster: {= cluster_name =}
+  data:
+    nova-compute-secrets.conf:{= " " =}
+      {%- filter b64enc %}
```

I guess you preferred having the `filter` start on a separate line instead of

```suggestion
    nova-compute-secrets.conf: {% filter b64enc %}
```
?

If so, then 👍🏾